aten/src/ATen/Context.cpp

```diff
@@ -332,8 +332,8 @@ const std::vector<at::QEngine>& Context::supportedQEngines() {
 
 #ifdef USE_FBGEMM
     if (fbgemm::fbgemmSupportedCPU()) {
-      // The X86 qengine is available if and only if FBGEMM is available
       engines.push_back(at::kX86);
+      // The X86 qengine is available if and only if FBGEMM is available
       engines.push_back(at::kFBGEMM);
```

Comment:
The order is not important now?

Reply:
maybe, I just revert @jerryzh168 's PR changes.

Reply:
the order is still important, whatever comes the last is the default, and we want to fix it in a separate PR